modules/release-notes/ref-release-notes-breaking-changes.adoc

```diff
@@ -23,9 +23,9 @@ The `scopes` parameter is now mandatory for the `gitlab:projectDeployToken:creat
 * link:https://issues.redhat.com/browse/RHIDP-5812[RHIDP-5812]
 
 [id="breaking-change-rhidp-5568"]
-== The `dynamicPlugins.cache.volumeClaimSpec` field is removed from the Helm Chart, dynamic plugin storage is now ephemeral
+== The `dynamicPlugins.cache.volumeClaimSpec` field is removed from the Helm Chart, dynamic plugin storage is now ephemeral by default
 
-The `dynamicPlugins.cache.volumeClaimSpec` field has been removed from `values.yaml`, making all dynamic plugin storage ephemeral by default. Instead of the Helm chart managing persistent volume claim (PVC) creation, you must now manually configure ephemeral volume claims under `extraVolumes`.
+The `dynamicPlugins.cache.volumeClaimSpec` field has been removed from `values.yaml`, making dynamic plugin storage ephemeral by default. If you want to enable plugin caching persistence, you must now manually configure a `PersistentVolumeClaim` and mount it using the `extraVolumes` and `extraVolumeMounts` fields in the Helm chart values.
```

Comment:
Would it make sense to also link to the actual doc on how to configure this?
https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.5/html/configuring_red_hat_developer_hub/con-dynamic-plugin-cache_running-behind-a-proxy#creating-a-pvc-for-the-dynamic-plugin-cache-using-the-helm-chart